cypresse2e/cypress/integration/e2eTests/header/header.spec.js

```diff
@@ -308,7 +308,13 @@ describe("FightPandemics Navigation bar", () => {
     //after clicking style is changed to bold
     language
       .click({ force: true })
-      .should("have.attr", "style", "font-weight: bold;");
+      .get(languageSelector)
+      .should(($languageEl) => {
+        expect($languageEl)
+          .to.have.attr("style")
+          .contains("font-weight: bold;");
+      });
```

Comment:
Below is another way to write the code with less text but your way works too

```
language
      .click({ force: true })
      .get(languageSelector)
      .should("have.attr", "style", "font-weight: bold;");
```
![image](https://user-images.githubusercontent.com/56090896/104390227-33621800-550b-11eb-9f07-c68678b09206.png)

Reply:
TY @jessica-lau

Reply:
I decided to fix it as recommened